scripts/deploy.sh

```diff
@@ -101,35 +99,82 @@ verify_proxy_deployer_module() {
     verify_module_contract "$1" "$MODULE_ADDR" proxy "src/Managed.sol:ManagedProxy"
 }
 
-deploy_create2_deployer() {
-    print_title "Sending funds to the address deploying CREATE2 deployer"
+deploy_deterministic_deployer() {
+    print_title "Sending funds to the address deploying deterministic deployer"
     # Taken from https://github.com/Arachnid/deterministic-deployment-proxy
     cast send $WALLET_ARGS --value "0.01ether" "0x3fAB184622Dc19b6109349B94811493BF2a45362"
 
-    print_title "Deploying the CREATE2 deployer"
+    print_title "Deploying deterministic deployer"
     # Taken from https://github.com/Arachnid/deterministic-deployment-proxy
     cast publish "0xf8a58085174876e800830186a08080b853604580600e600039806000f350fe7ffffffffffffffff\
```

Comment:
Wondering if we can make this not hard-coded. I just compared it with the bytecode in the readme.

Reply:
We probably need to have SOMETHING hardcoded. This is a pre-signed transaction payload, if we wanted to build it ourselves, we'd need to build a transaction with a hardcoded signature and bytecode. If we wanted to build the bytecode, we'd need to copy the YUL source and compile it with a specific version of solc. IMO it's not worth it, having a single, easily verifiable blob is probably the cleanest approach.